swagger.yaml

```diff
@@ -844,4 +1006,53 @@ components:
         lastUpdatedAt:
           type: string
           format: date-time
-          description: Date and time when the software entry was last updated
\ No newline at end of file
+          description: Date and time when the software entry was last updated
+        
+    Questionnaire:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: Unique identifier for the questionnaire
+        title:
+          type: string
+          description: Title of the questionnaire
+        description:
+          type: string
+          description: Description of the questionnaire
+        options:
+          type: array
+          items:
+            $ref: '#/components/schemas/Questions'
+          description: List of questions in the questionnaire
+        tags:
+          type: array
+          items:
+            type: string
+          description: List of tags for the questionnaire
+        passedUsers:
```

Comment:
I wonder if we can come up with a more descriptive name, I feel passedUsers is a bit vague. But I can't think of a better name so it can be left as is if you think this is the best option.